src/oceans/ui.jsx

```diff
@@ -1040,14 +1040,12 @@ let Predict = class Predict extends React.Component {
       this.state.timeScale !== defaultTimeScale
     ) {
       selectedControl = MediaControl.FastForward;
-    } else {
-      selectedControl = MediaControl.Play;
     }
 
     return (
       <Body>
         {this.state.displayControls && (
-          <div style={styles.mediaControls}>
+          <div style={styles.mediaControls} id="uitest-media-ctrl">
```

Comment:
Is there any way to avoid using DOM IDs, say by using React references to these elements?  Or is this the preferred technique?

Reply:
i was following the convention in the code-dot-org repo, which is to use IDs prefaced with 'uitest,' so i think this is the preferred technique